app/post/controller.js

```diff
@@ -16,6 +16,15 @@ module.exports.create = function (req, res) {
   });
 };
 
+module.exports.setHidden = function (req, res) {
+  Post.findById(req.params.id, function (err, post) {
+    Post.setHidden(post, function (err, writeResult){
+      if (err) { throw err; }
+      res.send(writeResult);
```

Comment:
Something needs to happen when a delete action is fired, and sending raw data to the user is certainly not good user experience. Consider a redirect and maybe a flash message.